### ISSUE TITLE
metgrid release-v4.0.2 IF test, should have 4.0.1, not 4.0.2 twice

### ISSUE THREAD

**Issue by davegill** [wrf-model/WPS#94]
Likely this transcription error is in master and develop also.

https://github.com/wrf-model/WPS/blob/fd38c2939a47e2f782d81c8206e6ea3badf91cc2/metgrid/src/input_module.F#L432

**Comment by mgduda** [comment]
Should we release a fix as v4.0.2.1?

**Comment by davegill** [comment]
@mgduda @weiwangncar @kkeene44 
Folks,
After testing, this is indeed a bug, but it has no impact.

Here's the code:
```
         call ext_get_dom_ti_char          ('TITLE', title)
         if (index(title,'GEOGRID V4.0.2') /= 0) then
            wps_version = 4.02
         else if (index(title,'GEOGRID V4.0.2') /= 0) then
            wps_version = 4.01
         else if (index(title,'GEOGRID V4.0') /= 0) then
            wps_version = 4.0
BLAH BLAH BLAH
         else if (index(title,'GEOGRID V3.1.1') /= 0) then
            wps_version = 3.11
         else if (index(title,'GEOGRID V3.1') /= 0) then
            wps_version = 3.1
         else if (index(title,'GEOGRID V3.0.1') /= 0) then
            wps_version = 3.01
         else
            wps_version = 3.0
         end if
         call mprintf(.true.,STDOUT,'Reading static data from WPS version %f', f1=wps_version)
```
We expect to fall into last ELSE with version 3.0, which would be a problem with this subsequent test:
```
         if ( wps_version >= 3.01 ) then
            call ext_get_dom_ti_integer_scalar('NUM_LAND_CAT', num_land_cat)
         else
            num_land_cat
         end if 
```
However, here's the print out that we get
```
Reading static data from WPS version 4.000000
```
The title string from geogrid is "GEOGRID V4.0.1". We do not find that string, but we do successfully 
accidentally find "GEOGRID V4.0". This is incorrect, but there is no side effect.

I used two different geogrid output files:
```
> ncdump -h v401/geo_em.d01.nc > v401.txt
> ncdump -h v402/geo_em.d01.nc > v402.txt
> diff v401.txt v402.txt
482c482
< 		:TITLE = "OUTPUT FROM GEOGRID V4.0.1" ;
---
> 		:TITLE = "OUTPUT FROM GEOGRID V4.0.2" ;
```
The generated metgrid files have identical fields and identical header files.
